load.php

```diff
@@ -39,5 +41,11 @@
 		'enabled' => get_environment_architecture() === 'chassis',
 	];
 
-	register_module( 'local-chassis', __DIR__, 'Local Chassis', $default_settings );
+	register_module(
+		'local-chassis',
+		__DIR__,
+		'Local Chassis',
+		$default_settings,
+		'Altis\\Chassis\\bootstrap'
```

Comment:
```suggestion
		__NAMESPACE__ . '\\bootstrap'
```

Reply:
This file is not under the `Altis\Chassis` namespace but the `Altis\Local_Chassis` namespace. This change would introduce a fatal error

Reply:
Yep, see the other review comment to change the namespace there.